src/base/BlockInterface.php

```diff
@@ -97,6 +97,20 @@ public function blockGroup();
      */
     public function getFieldHelp();
 
+    /**
+     * Set the block's {{luya\cms\models\NavItemPage}} object.
+     *
+     * @param NavItemPage $page The page object.
+     */
+    public function setPage($page);
+
+    /**
+     * Returns the block's {{luya\cms\models\NavItemPage}} object.
+     *
+     * @return NavItemPage
+     */
+    public function getPage();
```

Comment:
since we support php 8 only, we should add a getPage() : NavItemPage;

Reply:
Added.